static/chrome/entrypoint.sh

```diff
@@ -41,8 +41,14 @@ if env | grep -q ROOT_CA_; then
   for e in $(env | grep ROOT_CA_ | sed -e 's/=.*$//'); do
     certname=$(echo -n $e | sed -e 's/ROOT_CA_//')
     echo ${!e} | base64 -d >/tmp/cert.pem
-    certutil -A -n ${certname} -t "TCu,Cu,Tu" -i /tmp/cert.pem -d sql:$HOME/.pki/nssdb
+    certutil -A -n ${certname} -t "TC,C,T" -i /tmp/cert.pem -d sql:$HOME/.pki/nssdb
+    if cat tmp/cert.pem | grep -q "PRIVATE KEY"; then
+      AUXILIARY_PASS=123456
```

Comment:
Need to be able to read key password (`AUXILIARY_PASS=123456`) from external environment variable.

Reply:
The actual purpose of AUXILIARY_PASS is providing consistent fake password for _openssl -export_ and _pk12util -i_ commands - nothing more. The variable AUXILIARY_PASS is replaced with EMPTY_PASS in 54e2bc8, which could be a little more clear.
Btw, some _*.pem_ files could have password-protected private keys, and such files are not supported in current PR.